compose.yml

```diff
@@ -32,13 +31,13 @@ services:
             - MW_RECAPTCHA_SITE_KEY
             - MW_RECAPTCHA_SECRET_KEY
             - MW_ADMIN_USER
-            - MW_ADMIN_PASS
+            - MW_ADMIN_PASS_FILE=/run/secrets/mw_admin_pass
```

Comment:
There is no `MW_ADMIN_PASS_FILE` environment and nothing handles reading the password from file on the wiki container so this won't work

Reply:
I  did not know about that, great!